### ISSUE TITLE
App crashes when trying to consume item

### ISSUE THREAD

**Issue by draeklae** [patzly/grocy-android#426]
As the title says, the app crashes every time I try to consume an item. More specifically:

1. Open app
2. Select "Consume"
3. Type in product name
4. Select product from pop-up list
5. Crash

I can consume items without problems on the web app, so (I assume) this is not a consistency problem in my database.

App version: 2.0.0-beta02
Server version: 3.0.1
Android version: 11

Debug log:
```
06-19 14:23:13.362 10974 10974 E AndroidRuntime: FATAL EXCEPTION: main
06-19 14:23:13.362 10974 10974 E AndroidRuntime: Process: xyz.zedler.patrick.grocy, PID: 10974
06-19 14:23:13.362 10974 10974 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String xyz.zedler.patrick.grocy.model.StockLocation.i()' on a null object reference
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.model.Z0.j(SourceFile:13)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.I5.c0(SourceFile:2)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.fragment.ConsumeFragment.l2(SourceFile:6)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.fragment.ConsumeFragment.q2(SourceFile:15)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.fragment.g.a(Unknown Source:2)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.E.c(SourceFile:1)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.e6.o(SourceFile:2)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.w4.c(Unknown Source:4)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at androidx.lifecycle.LiveData.c(SourceFile:6)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at androidx.lifecycle.LiveData.d(SourceFile:8)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at androidx.lifecycle.LiveData.n(SourceFile:4)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at androidx.lifecycle.p.n(SourceFile:1)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.e6.n(SourceFile:2)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.H5.j(SourceFile:1)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.I5.R(SourceFile:35)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.l.t.a(Unknown Source:4)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.h.J0$I.d(SourceFile:4)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.h.h0.a(Unknown Source:2)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.h.m.a(SourceFile:8)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.h.a.a(Unknown Source:4)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.m.g.a(SourceFile:3)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at b.a.b.x.m.e(SourceFile:5)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at b.a.b.g$b.run(SourceFile:5)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:938)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
06-19 14:23:13.362 10974 10974 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by MrJohnZoidberg** [comment]
Could you please tell us whether the issue still exists and if yes, install the debug APK from below? Then you can capture & send us the log again – it will be more readable (it has minify and ProGuard disabled). Thanks!

**Comment by MrJohnZoidberg** [comment]
@draeklae ?

**Comment by draeklae** [comment]
Problem is fixed as of latest F-Droid version (2.0.0-beta03). Thanks so much!

**Comment by MrJohnZoidberg** [comment]
Great! But I didn't do anything to fix it... :-)
So the crash occurs again, feel free to reopen the issue or create a new one.
